logic-apps/pull-reference-data.json

```diff
@@ -608,6 +608,10 @@
                         },
                         "SensorId": {
                           "type": "string"
+                        },
+                        "JobLastRegistrationDateTime": {
+                          "format": "datetime",
+                          "type": "string"
```

Comment:
Why are we adding the new ones? They are not used from ASA queries (yet?).

Reply:
Yes, they are not used as of now.
But i recently made update to entities to bring them in sync in terms of fields they are exposing. So you can see that one field added in one schema is already there in other one and vice versa.
They will be required by our customers when they starting using these scenarios and start customizing them.